### PR TITLE
Header

### PR DESCRIPTION
**Pull request by KarthikRaju391**
### Following changes have been implemented

- Added the header component to App.js

### CODE REVIEW

**Comment by mentor-tara** [comment]
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime, consider starting with the next task already to practice the professional Git workflow. [If you need a refresher, feel free to follow this section in the course again.](https://profy.dev/project/reddit-timer/while-waiting)[](ckma5leps3kko0b87rxtjjp24)

**Review by mentor-tara — CHANGES_REQUESTED** [review]
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] Use absolute instead of relative links: `<Link to="/search" />` [:link:](https://profy.dev/project/reddit-timer/review-tips/use-absolute-links)
- [x] No components that solely copy props to their children [:link:](https://profy.dev/project/reddit-timer/review-tips/unnecessary-components)
- [x] Use human readable URLs [:link:](https://profy.dev/project/reddit-timer/review-tips/human-readable-links)
- [x] Optimize svg files with SVGOMG [:link:](https://profy.dev/project/reddit-timer/review-tips/optimize-svg-with-svgomg)
- [x] Use react-router Link instead of HTML anchor tag [:link:](https://profy.dev/project/reddit-timer/review-tips/link-instead-of-anchor-tag)

**Styles:**
- [x] **Where to put your styles** [:link:](https://profy.dev/project/reddit-timer/review-tips/where-to-put-styles)
- [x] **Refrain from styling children with CSS selectors** [:link:](https://profy.dev/project/reddit-timer/review-tips/css-selectors)
- [x] **Extract shared CSS values** [:link:](https://profy.dev/project/reddit-timer/review-tips/common-css-values-in-theme)
- [x] Be careful with pixel-perfectness when using relative font sizes like `0.9em` [:link:](https://profy.dev/project/reddit-timer/review-tips/careful-with-relative-font-size)

**Tests:**
- [x] **Use React Testing Library for testing** [:link:](https://profy.dev/project/reddit-timer/review-tips/use-react-testing-library)
- [x] Don't write snapshot tests [:link:](https://profy.dev/project/reddit-timer/review-tips/no-snapshot-tests)
- [x] **Prefer integration tests over unit tests** [:link:](https://profy.dev/project/reddit-timer/review-tips/no-unit-tests)
- [x] Get the logo by role with `getByRole` [:link:](https://profy.dev/project/reddit-timer/review-tips/get-logo-by-role)
- [x] How to test hash links [:link:](https://profy.dev/project/reddit-timer/review-tips/testing-hash-links)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/header-ae7e45f0-51b2-433a-b293-3bef0ae80cce). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckmaavvyw3m7p0b358cxn7dzq)

**Comment by jkettmann** [comment]
@KarthikRaju391 good job. Let me know if you're stuck with the merge conflicts 😄

**Review by mentor-tara — APPROVED** [review]
Great work! :tada::tada::tada: You can merge the PR now.[](ckmabg2tk3mkv0b35j06077kr)

**Comment by KarthikRaju391** [comment]
Yeah, I had some confusion, but I merged the conflicts today.
I don't know if I can ask my doubts here but I am having issues with ESLINT
when I try to do the 'git add .' command.
Something like 'warning: LF will be replaced by CRLF in
src/Components/header.js.'
Sorry if I am asking too much.
Thank you for your time.

**Comment by jkettmann** [comment]
You're on Windows, right? This is an issue with the end of line. According to [stackoverflow](https://stackoverflow.com/questions/5834014/lf-will-be-replaced-by-crlf-in-git-what-is-that-and-is-it-important) this Git setting will help. Just type it into your terminal

```
git config core.autocrlf true
```

Thanks for asking btw. It's always good to know what people are struggling with 😉

File: src/Components/header.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  }
+`;
+
+const ListItem = styled.li`
+  text-decoration: none;
+`;
+
+const NavLinks = styled.ul`
+  list-style-type: none;
+  font-family: 'Montserrat', sans-serif;
```
Comment (jkettmann) [inline src/Components/header.js:28]: Again, very clean code. Nice job. 

I would set the font-family in the global styles. I didn't run the project but I assume most of the elements have a default font now, right? This should be Montserrat though iirc

File: src/Components/header.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const Container = styled.header`
+  max-width: 1300px;
+  width: 80%;
```
Comment (jkettmann) [inline src/Components/header.js:7]: I'd rather use `width: 100%` here with a fixed horizontal padding. I guess it's 80px in the design. Using `width: 80%` will make it hard to align other elements with the header and might look strange on smaller devices. But that's just my guess. 

I usually see `width: 100%` in combination with a `max-width` and some padding. You can also check the example implementation.

File: src/Components/header.js (hunk 1 above)
Comment (KarthikRaju391) [reply src/Components/header.js:28]: This is the first time I am working with Styled-components. I forgot to put the fonts in GlobalStyles. Anyway, I'll make the changes now. Thank you.

File: src/Components/header.js (hunk 2 above)
Comment (KarthikRaju391) [reply src/Components/header.js:7]: I have fixed it now. These are some really helpful insights. 
I'm absolutely enjoying this course!

**Comment by KarthikRaju391** [comment]
I have implemented all these changes, please check it out when you have
time, thank you.

**Pull request merged**